ci-operator/config/quay/quay-tests/quay-quay-tests-master__ocp414-quay38.yaml

```diff
@@ -37,7 +37,8 @@ tests:
       FIREWATCH_CONFIG: |
         [
           {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "OCPQE"},
-          {"step": "gather-*", "failure_type": "all", "classification": "Other", "jira_project": "OCPQE"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "OCPQE"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
           {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "OCPQE"},
           {"step": "enable-*", "failure_type": "all", "classification": "Other", "jira_project": "OCPQE"},
           {"step": "quay-tests-deploy-*", "failure_type": "all", "classification":  "Quay deployment", "jira_project": "OCPQE"},
```

Comment:
If the Quay test was failed, suggest file new jira ticket to project "PROJQUAY" rather than OCPQE Project.